sdk/eventhub/sample.env

```diff
@@ -17,4 +17,4 @@ DOMAIN="<tenant id>"
 AZURE_SUBSCRIPTION_ID="<subscription id>"
 
 # For EventProcessorHost (EPH).
-STORAGE_CONNECTION_STRING="DefaultEndpointsProtocol=https;AccountName=<storage account name>;AccountKey=<account key>;EndpointSuffix=core.windows.net"
+STORAGE_CONNECTION_STRING="DefaultEndpointsProtocol=https;AccountName=<storage account name>;AccountKey=<account key>;EndpointSuffix=core.windows.net"
```

Comment:
Sorry, I tried leaving this comment earlier today but GitHub was having some issues. This file shouldn't have any changes. Can you revert the changes to this file?

Reply:
yes please